src/components/molecules/MoleculeTestimonialCard.vue

```diff
@@ -44,7 +44,7 @@ export default {
 <style lang="scss" scoped>
 $g-line-height: 1.5 !default;
 $g-spacing: $g-line-height * 1em;
-$black: #000;
+$black: var(--primary-text);
```

Comment:
why can't we directly use var(--primary-text) instead of using $black
because if tomorrow we change primary-text color, black value would change which shouldn't be ideal.

Reply:
done